### ISSUE TITLE
Discord service does not work

### ISSUE THREAD

**Issue by mbrandau** [containrrr/shoutrrr#15]
1. The documentation in the README is wrong. It should be `discord://token@channel` instead of `discord://channel/token`
2. Somehow the URL is not correctly parsed and the config values for `Channel` and `Token` are empty

```bash
./shoutrrr generate discord://awdawdawd@awdawdawd
Service: discord://awdawdawd@awdawdawd
Channel: Token: %
```

I would suggest to add tests for each service to check if the URL is parsed and correctly stored in the config.

**Comment by simskij** [comment]
Thanks! Will have a look at this!

**Comment by simskij** [comment]
Solved in 6e7a09129f77d584620e2047562221a5cefdf684
